ui/src/main/scripts/plugins/e2e/ts/e2e.ts

```diff
@@ -77,7 +77,7 @@ module E2E {
       $scope.operations = [];
 
       _.each($scope.e2eData, (node) => {
-        $scope.topLevel.push(node.id);
+        $scope.topLevel.push(node.serviceName + ': ' + node.id);
```

Comment:
good point https://github.com/hawkular/hawkular-apm/pull/585/files needs to go first, then I will change this to dealt with no service name.